### PR TITLE
Longrun output summary

### LINKED ISSUE #721: Enhance longrun output plot summary
# build

https://buildkite.com/clima/climacoupler-longruns/builds/558

# components
- [x] rm slabplanet conservation logs
- [x] NCEP and AMIP plots comparison on one page
- [x] apply leadership board in AMIP longruns (path fixed and added seasonal)
- [x] ~build_history as a direct link to Buildkite~
- [x] make all runs soft fail (here hard fails are not necessary and they can stop the slack report from other passing runs)
- [x] add an array of dates to bias plots
- [x] Error 26 (this only happened for a 2 hour period and worked after retry)
- [x] rm scaling tests - these will be moved to a separate pipeline (on GPU) #663 

## Buildkite URL issue
- The URL link pass can work but it looks like there is a time limit for how long it is accessible:

```
      - label: ":envelope: Slack report: build_history new"
        command:
          - |
            ARTIFACT_ID=$(curl -s -H "Authorization: Bearer $BUILDKITE_READ_ARTIFACTS_API_TOKEN" -X GET "https://api.buildkite.com/v2/organizations/clima/pipelines/climacoupler-longruns/builds/$$BUILDKITE_BUILD_NUMBER/artifacts" | jq -r '.[] | select(.filename == "build_history.html") | .id')
            JOB_ID=$(curl -s -H "Authorization: Bearer $BUILDKITE_READ_ARTIFACTS_API_TOKEN" -X GET "https://api.buildkite.com/v2/organizations/clima/pipelines/climacoupler-longruns/builds/$$BUILDKITE_BUILD_NUMBER/artifacts" | jq -r '.[] | select(.filename == "build_history.html") | .job_id')
            URL=$(curl -H "Authorization: Bearer $BUILDKITE_READ_ARTIFACTS_API_TOKEN"   -X GET "https://api.buildkite.com/v2/organizations/clima/pipelines/climacoupler-longruns/builds/$$BUILDKITE_BUILD_NUMBER/jobs/$$JOB_ID/artifacts/$$ARTIFACT_ID/download"| jq -r '.url')
            echo "Interactive overall job performance history $$URL"
            slack-message -c "#coupler-report" -x ":rocket: Interactive overall job performance history <$${URL}|build_history> :rocket:"
```

-  When viewing an artifact on  Buildkite, a new URL is generated each time the artifact is being read, and each URL is only valid for 60s (see [here](https://buildkite.com/docs/apis/rest-api/artifacts#download-an-artifact)). For a longer lasting URL, we'd need to set up an S3 bucket (see [here](https://buildkite.com/docs/pipelines/artifacts#storage-providers-encryption-and-retention)) but it was decided this would not be worth the time investment.

### PR DESCRIPTION
**Pull request by LenkaNovak**
closes #721 

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -283,105 +294,13 @@ steps:
...
           slurm_gpus: 1
           slurm_mem: 16GB
+        soft_fail: true
 
-  - group: "Other AMIP targets"
-
-    steps:
-
-      # DYAMOND AMIP: 1 day (convection resolving)
-      - label: "MPI AMIP SUPERFINE: dyamond_target"
```
Comment (juliasloan25) [inline .buildkite/longruns/pipeline.yml:292]: are we still interested in DYAMOND, or not anymore?

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -394,21 +313,20 @@ steps:
...
-          - slack-upload -c "#coupler-report" -f experiments/AMIP/output/slabplanet/slabplanet_coupler_sf_evolve_ocn_artifacts/total_water_log_bucket.png -m png -n slab_coarse_w_log -x "Slabplanet water conservation (log error)"
           - slack-upload -c "#coupler-report" -f experiments/AMIP/output/slabplanet/slabplanet_coupler_sf_evolve_ocn_artifacts/total_water_bucket.png -m png -n slab_coarse_w -x "Slabplanet water conservation"
 
       - label: ":envelope: Slack report: target AMIP"
         command:
-          - slack-upload -c "#coupler-report" -f experiments/AMIP/output/amip/amip_target_artifacts/amip_paperplots.png -m png -n amip_fine -x "AMIP Target Longrun"
-          - slack-upload -c "#coupler-report" -f experiments/AMIP/output/amip/amip_target_artifacts/biases.png -m png -n amip_fine -x "AMIP Target Longrun"
+          - slack-upload -c "#coupler-report" -f experiments/AMIP/output/amip/amip_target_topo_diagedmf_artifacts/amip_ncep.png -m png -n amip_fine -x "Target AMIP v NCEP Last Month Mean"
```
Comment (juliasloan25) [inline .buildkite/longruns/pipeline.yml:326]: This plot looks pretty distorted on slack. Can we change the dimensions of the plots somehow?

image

**Review by juliasloan25 — COMMENTED** [review]
Thanks Lenka! I left a couple of comments - mostly I think the formatting of the AMIP/NCEP plots became a little compressed on the most recent build (I copied the image in one of my comments)

File: .buildkite/longruns/pipeline.yml (hunk 1 above)
Comment (LenkaNovak) [reply .buildkite/longruns/pipeline.yml:292]: We are, but the gpu version (`gpu_dyamond_target`) should be sufficient here.

File: .buildkite/longruns/pipeline.yml (hunk 2 above)
Comment (LenkaNovak) [reply .buildkite/longruns/pipeline.yml:326]: Yeah, I saw that. Something went wrong after the rebase. I'm pushing a more robust layout setup rn. :)

**Review by juliasloan25 — APPROVED** [review]

**Pull request merged**